vimdoc/regex.py

```diff
@@ -211,14 +211,23 @@ def _DelimitedRegex(pattern):
   # MATCH GROUP 1: The Name
   (
     # Non-commas or escaped commas or escaped escapes.
-    (?:[^\\,]|\\.)+
+    # Must not end with a space.
+    (?:[^\\,<]|\\.)+\S
   )
   # Optional identifier
   (?:
     # Separated by comma and whitespace.
-    ,\s*
+    \s*,\s*
```

Comment:
yes, there was an issue where the regexp was slurping spaces after the name in the case "name < parent", so i enforced the name ending with a nonspace and then i figured we might as well allow whitespaces around both the , and < separators for consistency.